pkg/signup/service/signup_service.go

```diff
@@ -612,58 +614,51 @@ func (s *ServiceImpl) PhoneNumberAlreadyInUse(userID, username, phoneNumberOrHas
 	return nil
 }
 
-func GetDefaultUserNamespace(provider ResourceProvider, signup signup.Signup) string {
-	sbSelector, err := labels.NewRequirement(toolchainv1alpha1.SpaceBindingMasterUserRecordLabelKey, selection.Equals, []string{signup.CompliantUsername})
-	if err != nil {
-		log.Errorf(nil, err, "unable to create spacebindings selector for signup %s", signup.Name)
-		return ""
-	}
-
-	requirements := []labels.Requirement{*sbSelector}
-
-	sbs, err := provider.ListSpaceBindings(requirements...)
-	if err != nil {
-		log.Errorf(nil, err, "unable to list spacebindings for signup %s", signup.Name)
-		return ""
-	}
-
-	// iterate through the SpaceBindings to find the Spaces that the user has access to, then look for the default namespace with a preference for a Space created by the user
-	var defaultNamespace string
-	var createdSpaceFound bool
-	for _, sb := range sbs {
-		spaceName := sb.Labels[toolchainv1alpha1.SpaceBindingSpaceLabelKey]
-		if spaceName == "" { // space may not be initialized
-			// log error and continue so that the api behaves in a best effort manner
-			log.Errorf(nil, fmt.Errorf("spacebinding has no '%s' label", toolchainv1alpha1.SpaceBindingSpaceLabelKey), "unable to get space '%s'", spaceName)
-			continue
-		}
-		space, err := provider.GetSpace(spaceName)
+func GetDefaultUserTarget(provider ResourceProvider, spaceName, murName string) (string, string) {
```

Comment:
Can you add a comment to the function stating that it returns `<target-member-cluster>, <default-namespace>`?

Reply:
fac5e9e